lib/internal/fs/streams.js

```diff
@@ -186,7 +205,7 @@ ReadStream.prototype._read = function(n) {
 
   // the actual read.
   this[kIsPerformingIO] = true;
-  fs.read(this.fd, pool, pool.used, toRead, this.pos, (er, bytesRead) => {
+  this[kFs].read(this.fd, pool, pool.used, toRead, this.pos, (er, bytesRead) => {
```

Comment:
@ronag The linter complains about this line being too long :/

Reply:
damn, what's the best way to break this line?

Reply:
this feels a bit silly, but I can't find in the style guidelines how Node prefers to break long function calls?

Reply:
Does `make lint-js-fix` do anything for you? Otherwise, I’d probably just break the line after the first `(` and indent by two spaces

Reply:
lint-js-fix didn't do much, I followed your suggestion, I don't like it but I don't have any better suggestion. Fixed.